src/libraries/System.Text.RegularExpressions/tests/RegexParserTests.cs

```diff
@@ -383,7 +383,7 @@ static RegexParserTests()
         [InlineData(@"[a-\c>]", RegexOptions.None, RegexParseError.UnrecognizedControl)]
         [InlineData(@"[a--]", RegexOptions.None, RegexParseError.ReversedCharRange)]
         [InlineData(@"[--a]", RegexOptions.None, null)]
-        [InlineData(@"[a-\-]", RegexOptions.None, null)]
+        [InlineData(@"[a-\-]", RegexOptions.None, RegexParseError.ReversedCharRange)]
```

Comment:
Given all of the churn in regex right now, I re-enabled the tests to run against netfx, to help minimize the chances of breaks.  As such, these tests that change expected results will need to be factored out into a separate test that is skipped on netfx.  Sorry for the hassle :)